CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ and this project adheres to [Semantic Versioning](http://semver.org).
 ## Supported Release [4.3.0]
```

Comment:
Needs updated.

Reply:
D'oh! How did I miss that! Good catch!

Reply:
I would also include the link at the bottom of the file eg:

[4.4.0]:https://github.com/puppetlabs/puppetlabs-apt/compare/4.3.0...4.4.0

Reply:
Good call, I'll keep an eye to do that in future